frontend/src/assets/config.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Remove the file from the repo?

Reply:
No, this is the default config for running locally. We'll overwrite this file with a mount for deployment